catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -448,8 +453,7 @@ private String updateColumnCommentFieldDefinition(
     JdbcColumn updateColumn =
         new JdbcColumn.Builder()
             .withName(col)
-            // TODO #1531 will add default value.
-            .withDefaultValue(null)
+            .withDefaultValue(column.defaultValue())
```

Comment:
does pg need same logic?

Reply:
Unnecessary. See comment https://github.com/datastrato/gravitino/blob/99daef41df1fddc8c1498533943a90b1a9b94189/catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcColumn.java#L29-L34

Reply:
It's unnecessary for PG, because PG can update column property without a completed definition